backend/census_historical_migration/sac_general_lib/audit_information.py

```diff
@@ -0,0 +1,150 @@
+from census_historical_migration.workbooklib.census_models.census import (
+    CensusGen22 as Gen,
+    CensusCfda22 as Cfda,
+    CensusFindings22 as Finding,
+)
+from census_historical_migration.base_field_maps import FormFieldMap, FormFieldInDissem
+from census_historical_migration.sac_general_lib.utils import (
+    _create_json_from_db_object,
+)
+import audit.validators
+
+DOLLAR_THRESHOLD = 750000
+
+mappings = [
+    FormFieldMap(
+        "dollar_threshold", None, FormFieldInDissem, DOLLAR_THRESHOLD, int
+    ),  # FIXME: There is no in_db mapping ?
+    FormFieldMap(
+        "gaap_results", None, FormFieldInDissem, [], list
+    ),  # FIXME: There is no in_db mapping?
+    FormFieldMap(
+        "is_going_concern_included", "goingconcern", FormFieldInDissem, None, bool
+    ),
+    FormFieldMap(
+        "is_internal_control_deficiency_disclosed",
+        "materialweakness",
+        FormFieldInDissem,
+        None,
+        bool,
+    ),
+    FormFieldMap(
+        "is_internal_control_material_weakness_disclosed",
+        "materialweakness_mp",
+        FormFieldInDissem,
+        None,
+        bool,
+    ),
+    FormFieldMap(
+        "is_material_noncompliance_disclosed",
+        "materialnoncompliance",
+        FormFieldInDissem,
+        None,
+        bool,
+    ),
+    FormFieldMap(
+        "is_aicpa_audit_guide_included",
+        "reportablecondition",
+        FormFieldInDissem,
+        None,
+        bool,
+    ),
+    FormFieldMap(
+        "is_low_risk_auditee", None, FormFieldInDissem, False, bool
+    ),  # FIXME: There is no in_db mapping?
+    FormFieldMap(
+        "agencies", None, "agencies_with_prior_findings", [], list
+    ),  # FIXME: There is no in_db mapping?
+]
+
+
+def _get_agency_prefixes(dbkey):
+    agencies = {}
+    cfdas = Cfda.select().where(Cfda.dbkey == dbkey)
+    cfda: Cfda
+    for cfda in cfdas:
+        agencies[int((cfda.cfda).split(".")[0])] = 1

```

Comment:
It might be this can be a bit cleaner, and perhaps use a `set` instead of the weird use of a hash table to simulate a set. Or, you can leave it. I know I wrote that code, and perhaps it can be clearer as to its purpose. Up to you.